src/mobu/services/monkey.py

```diff
@@ -216,15 +187,22 @@ async def run_once(self) -> str | None:
         """
         self._state = MonkeyState.RUNNING
         error = None
-        try:
-            await self.business.run_once()
-            self._state = MonkeyState.FINISHED
-        except Exception as e:
-            msg = "Exception thrown while doing monkey business"
-            self._logger.exception(msg)
-            error = str(e)
-            self._state = MonkeyState.ERROR
-        return error
+        with sentry_sdk.isolation_scope():
+            sentry_sdk.set_user({"username": self._user.username})
```

Comment:
If we have this, do we need the `gafaelfawr_user` thing that I asked about earlier?